crate/tests/test.bats

```diff
@@ -0,0 +1,8 @@
+source "${BATS_TEST_DIRNAME}/../plan.sh"
+
+@test "Version matches" {
+  mkdir "$(hab pkg path $TEST_ORIGIN/crate)/crate/logs"
```

Comment:
You can use `$HAB_ORIGIN` instead of `$TEST_ORIGIN` .   It should always be set inside the studio, which is where these tests are intended to run.